### DISCUSSION TITLE
Error with no domain specified

### DISCUSSION THREAD

**Discussion by GenesisAdam** [kereis/traefik-certs-dumper#38] [Q&A] [answered]
I am getting this error when I omitted the DOMAIN variable.
![2021-03-18 15_47_31-skyline - Opera](https://user-images.githubusercontent.com/61323470/111655878-ea2b8e80-8801-11eb-831e-bb1f5fb905ae.png)

The instruction allows for no domain.
Thanks.

Femi

**Comment by kereis** [comment, marked as answer]
Hey,

I took a look at the code and realized that even though checking the case where `DOMAIN` is empty, I probably forgot to remove the sanity check where the program would exit if `DOMAIN` is empty.

https://github.com/kereis/traefik-certs-dumper/blob/d2f15d1e0b2c2efbfe83550558e2b76ecd1a1313/bin/dump.sh#L334

We simply have to remove this check and rewrite it to only split the `DOMAIN` CSVs if `DOMAIN` is defined.

If you want, you can try to fix it and send in a PR. Otherwise I fix it as soon as possible. Thanks for pointing this out!

**Reply by GenesisAdam** [reply]
Thanks for your response.
I am not a programmer so won't be able to make the changes.

**Reply by kereis** [reply]
No problem. I take care of it :)

**Reply by kereis** [reply]
Did you try to pull the latest image? I took a look at the code again and actually, it should just log this:

```
Environment variable DOMAIN empty. Will dump all certificates possible...
```
